repos.d/nixos.yaml

```diff
@@ -1,20 +1,20 @@
 ###########################################################################
 # NixOS packages
 ###########################################################################
-- name: nix_stable  # XXX: name the next one as e.g. nix_stable_21_05
+- name: nix_stable_21_11
   type: repository
-  desc: nixpkgs stable 21.05
+  desc: nixpkgs stable 21.11
   statsgroup: nix
   family: nix
-  ruleset: [nix,nix_old_node_naming]  # don't set nix_old_node_naming for next release
+  ruleset: [nix]  # don't set nix_old_node_naming for next release
   color: '7eb2dd'
-  minpackages: 60000
+  minpackages: 70000
```

Comment:
Wait, are you sure?  For unstable it shows less ATM: https://repology.org/repositories/statistics/total

Reply:
On the whole I can't see motivation to try driving this very close to the real number anyway, but maybe there is some.

Reply:
>  For unstable it shows less ATM in the "total" column

This column reports a number of projects (aggregate entities), not individual packages. See [packages](https://repology.org/repositories/packages) tab or [parse logs](https://repology.org/log/9555990) in `updates` tab instead.

> On the whole I can't see motivation to try driving this very close to the real number anyway, but maybe there is some.

The purpose for this knob is to catch cases where upstream suddenly generates broken incomplete dump (and there were such cases on practice) so it makes sense to keep it close. It's not very important though.